Apps/phnetskope/netskope.json

```diff
@@ -1,1163 +1,1902 @@
 {
-    "appid": "d1b4a23f-f4a0-411e-9cea-b9534d542e01",
-    "name": "Netskope",
-    "description": "This app integrates with the Netskope to execute various investigative and polling actions",
-    "type": "endpoint",
-    "product_vendor": "Netskope",
-    "logo": "logo_netskope.svg",
-    "logo_dark": "logo_netskope_dark.svg",
-    "product_name": "Netskope",
-    "product_version_regex": ".*",
-    "publisher": "Phantom",
-    "license": "Copyright (c) 2018-2019 Splunk Inc.",
-    "app_version": "1.0.4",
-    "utctime_updated": "2019-01-11T00:54:10.000000Z",
-    "package_name": "phantom_netskope",
-    "main_module": "netskope_connector.pyc",
-    "min_phantom_version": "4.0.1068",
-    "app_wizard_version": "1.0.0",
-    "configuration": {
-        "server_url": {
-            "description": "Server URL",
-            "data_type": "string",
-            "required": true,
-            "order": 0
-        },
-        "api_key": {
-            "description": "API Key",
-            "data_type": "password",
-            "required": true,
-            "order": 1
+  "appid": "d1b4a23f-f4a0-411e-9cea-b9534d542e01",
```

Comment:
It looks like all of the spacing was changed in this JSON file. Is there a way that you can push your changes to the JSON file so that it only shows what was changed/what you added?

Reply:
Nope, unfortunately, it got formatted, and I don't think we can revert the formatting without losing work.

Reply:
I would try to reformat it to match the original spacing.

Reply:
Example of what the JSON changes should look like: https://github.com/phantomcyber/phantom-apps/pull/158/files#diff-1d7ee31685866992ceff6f65eeef371c

Reply:
I don't think I have the original spacing. The src was emailed through a few hands before getting to mine. Moving forward, this is what will be done, but for now, I don't think I have what you need.

Reply:
Original spacing here: https://github.com/phantomcyber/phantom-apps/blob/next/Apps/phnetskope/netskope.json